src/utils.js

```diff
@@ -1,3 +1,22 @@
+export const refreshTokenSetup = (res) => {
```

Comment:
This should go below `SeasonNaming` so that we can keep constants and functions separate